packages/php-platform/.github/workflows/rebuild-dev-image.yaml

```diff
@@ -0,0 +1,33 @@
+on:
+    schedule:
+        -   cron: '0 3 * * *'
+    workflow_dispatch:
+name: Rebuild PHP Platform Docker image
+jobs:
+    rebuild-php-fpm-images:
+        name: Rebuild ${{ matrix.branches }} branch
+        runs-on: ubuntu-22.04
+        strategy:
+            matrix:
+                branches: [ '13.0' ]
```

Comment:
true, but that would require backporting this change into the previous versions. 

I did not find it necessary because: 
- As versions 11 and 12 are not developed actively, but merely bug-fixed, I think it's a waste of computer resources to periodically build images for those branches
- To leverage the benefits to the end-user it would require us to release a version – and 11 is now only bug-fixed

Generally speaking, the 13.0 version of the base image may be used even for the 11.0 and the 12.0 versions 🤔